.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-7.7.0
+6.0.1
```

Comment:
Would be good to use the same version, to rule out spurious diffs introduced by a changed version.

[Looks like](https://github.com/OpenAPITools/openapi-generator) 7.12.0 is latest, atm, but I'd suggest changing the version as a separate PR

Reply:
What about just using the same version that was previously used? (7.7.0)

Just trying to rule out introducing spurious changes from going backwards on the version here.